scripts/makeKeySecret.sh

```diff
@@ -9,6 +9,10 @@ echo 'where <key-id> can be obtained from `gpg -k --key-id-format=long`'
 fi
 shift
 
+if [ "$*" == "" ]; then
```

Comment:
what's this?

Reply:
its a minor enhancement sliding in under the covers 
it reminds a consumer of this totally optional, just for help, script that if they dont say what key they want they get them all